### PR TITLE
Aeyb #25 roles page

### PR DESCRIPTION
**Pull request by grantliunz**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -12,6 +13,7 @@ function App() {
                 <Routes>
                     <Route path="/" element={<Login />} />
                     <Route path="*" element={<NotFound />} />
+                    <Route path="roles" element={<Roles />} />
```
Comment (TylerDonghwi) [inline frontend/src/App.tsx:17]: did you pull the main into your branch after I merged my homepage?

**Review by TylerDonghwi — CHANGES_REQUESTED** [review]
I think you just need to merge the main into your branch so you're up to date with the current branch

File: frontend/src/App.tsx (hunk 1 above)
Comment (grantliunz) [reply frontend/src/App.tsx:17]: done

**Review by TylerDonghwi — COMMENTED** [review]
LGTM now!!

**Review by TylerDonghwi — APPROVED** [review]

**Pull request merged**